test/cpp/jit/test_alias_analysis.h

```diff
@@ -997,13 +1017,13 @@ void testMemoryDAG() {
 void testAliasRegistration() {
   {
     auto registry = torch::RegisterOperators().op(
-        "foo::rand",
+        "foo::rand1",
```

Comment:
What was causing alias analysis settings to persist globally and non-rewritable? Can we fix the underlying issue instead of making all the tests cases happen to be uniquely name?

Reply:
The issue is that JIT doesn't allow deregistration of operators. So once registered, always registered, and the aliasing info remains. This isn't the only place where this is causing issues, but fixing that is out of scope for this diff.